Bottom: bd7fc6913845a91ab87801e3e2f767bfe26a4c86
Top:    c3c3b93b59a2b21865227ff9c4f1d9701b20537e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 17:41:25 +0000

Refresh of attempt-to-refactor-the-step

---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index 870afc4..0c197da 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -11,7 +11,6 @@
 #include "hid/common/face3d.h"
 #include "hid/common/edge3d.h"
 #include "hid/common/object3d.h"
-#include "polygon.h"
 #include "data.h"
 
 #include "step_writer.h"
